Simplenote/src/androidTest/java/com/automattic/simplenote/utils/EspressoUITests.java

```diff
@@ -0,0 +1,82 @@
+package com.automattic.simplenote.utils;
+
+import android.app.Instrumentation;
+import android.os.SystemClock;
+import android.view.MotionEvent;
+
+import com.automattic.simplenote.NotesActivity;
+import com.automattic.simplenote.R;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+import androidx.test.rule.ActivityTestRule;
+
+import static androidx.test.platform.app.InstrumentationRegistry.getInstrumentation;
+import static com.schibsted.spain.barista.interaction.BaristaClickInteractions.clickOn;
+import static com.schibsted.spain.barista.interaction.BaristaDrawerInteractions.openDrawer;
+import static com.schibsted.spain.barista.interaction.BaristaEditTextInteractions.writeTo;
+import static com.schibsted.spain.barista.interaction.BaristaListInteractions.clickListItem;
+import static java.lang.Thread.sleep;
+
+public class EspressoUITests {
+
+    @Rule
+    public ActivityTestRule<NotesActivity> mActivityTestRule = new ActivityTestRule<>(NotesActivity.class);
+
+    @Test
+    public static void loginWithCredentials() throws InterruptedException {
+        clickOn(R.id.button_login);
+        clickOn(R.id.button_email);
+        writeTo(R.id.input_email, "test.espresso.00001@gmail.com");
+        writeTo(R.id.input_password, "*-Re7]J4Ux8q)g?X");
```

Comment:
We should try to avoid putting valid credentials directly in the tests; could you move these to the `build.gradle` file so they can be added individually in `Simplenote/gradle.properties`? (Similar to other keys and credentials used in the app.)

Reply:
updated, thanks! 😄